packages/widget/src/components/transfer/fungible/transfer-detail/transfer-detail.ts

```diff
@@ -0,0 +1,87 @@
+import { FeeHandlerType } from '@buildwithsygma/sygma-sdk-core';
+import type {
+  Config,
+  Domain,
+  EvmFee,
+  Resource
+} from '@buildwithsygma/sygma-sdk-core';
+import '../../../common/buttons/button';
+import type { HTMLTemplateResult } from 'lit';
+import { html } from 'lit';
+import { when } from 'lit/directives/when.js';
+import { customElement, property } from 'lit/decorators.js';
+import { tokenBalanceToNumber } from '../../../../utils/token';
+import { BaseComponent } from '../../../common/base-component';
+import { styles } from './styles';
+
+@customElement('sygma-fungible-transfer-detail')
+export class FungibleTransferDetail extends BaseComponent {
+  static styles = styles;
+
+  @property({ type: Object })
+  fee?: EvmFee;
+
+  @property({ type: Object })
+  selectedResource?: Resource;
+
+  @property({ type: Object })
+  sourceNetwork?: Domain;
+
+  @property({ type: Object })
+  config?: Config;
```

Comment:
maybe you can replace config and sourceNetwork properties with DomainConfig property?